src/anastomosis.jl

```diff
@@ -83,10 +84,17 @@ end
 
 Return the Riemann invariants at the anastomosis node.
 """
-function calculateWstarAnastomosis(U, k)
-    @fastmath @inbounds W1 = U[1] + 4*k[1]*U[4]
-    @fastmath @inbounds W2 = U[2] + 4*k[2]*U[5]
-    @fastmath @inbounds W3 = U[3] - 4*k[3]*U[6]
+function calculateWstarAnastomosis(U, k, vessels :: Array{Vessel,1})
+	# MODIFIED THIS FUNCTION
+    v1 = vessels[1]						# MODIFIED THIS LINE
+    v2 = vessels[2]						# MODIFIED THIS LINE
+    v3 = vessels[3]						# MODIFIED THIS LINE
+    @fastmath @inbounds g1 = sqrt(1-v1.Ac/U[4]^4)		# MODIFIED THIS LINE
```

Comment:
in generale `*` é più veloce di `^`, quindi puoi riscrivere come `v1.Ac/(U[4]*U[4]*U[4]*U[4])`

in generale se usi direttamente `1.0` invece di `1`, eviti la conversione durante l'esecuzione